utils/api/api.config.ts

```diff
@@ -31,6 +31,10 @@ export const getApiUri = () => {
   // Get configured dev API URI
   const envApiUri = process.env.EXPO_PUBLIC_DEV_API_URI;
 
+  if (!envApiUri) {
```

Comment:
We should move this env validation to the very start of the application using zod to parse and derive types for env.d.ts